.github/workflows/test.yml

```diff
@@ -0,0 +1,48 @@
+name: Tests
+
+on:
+  push:
+    branches: [ main, master ]
```

Comment:
main and master - are two different approaches to name the trunk branch so it's better to choose on of them instead of pointing to both of them pretending on generic workflow.